hibernate-core/src/main/java/org/hibernate/dialect/Dialect.java

```diff
@@ -2957,6 +2958,17 @@ public String inlineLiteral(String literal) {
 		return String.format( "\'%s\'", escapeLiteral( literal ) );
 	}
 
+	/**
+	 * Check whether the JDBC {@link java.sql.Connection} supports creating LOBs via {@link Connection#createBlob()} or {@link Connection#createBlob()}.
+	 *
+	 * @param databaseMetaData JDBC {@link DatabaseMetaData} which can be used in case only starting from a given Driver version LOB creation is supported.
```

Comment:
`used in case only starting from a given Driver version LOB creation is supported`

I think I would go with:

`used if LOB creation is supported only starting from a given Driver version`